sdk/communication/Azure.Communication.PhoneNumbers/src/Azure.Communication.PhoneNumbers.csproj

```diff
@@ -7,7 +7,7 @@
     <AssemblyTitle>Azure Communication Phone Numbers Service</AssemblyTitle>
     <Version>1.2.0-beta.2</Version>
```

Comment:
Should we bump to a new beta version?

Reply:
will update the version while creating a new release